pages/public_cloud/containers_orchestration/managed_kubernetes/configuring-multi-attach-persistent-volumes-with-ovh-efs/guide.fr-fr.md

```diff
@@ -0,0 +1,409 @@
+---
+title: Configuring multi-attach persistent volumes with OVHcloud EFS
+excerpt: 'Find out how to configure a multi-attach persistent volume using OVHcloud EFS'
+updated: 2024-12-16
+---
+
+<style>
+ pre {
+     font-size: 14px;
+ }
+ pre.console {
+   background-color: #300A24;
+   color: #ccc;
+   font-family: monospace;
+   padding: 5px;
+   margin-bottom: 5px;
+ }
+ pre.console code {
+   border: solid 0px transparent;
+   font-family: monospace !important;
+   font-size: 0.75em;
+   color: #ccc;
+ }
+ .small {
+     font-size: 0.75em;
+ }
+</style>
+
+## Objective
+
+OVHcloud Managed Kubernetes currently offers Block Storage for persistent volumes by default, but that may not be suited for applications that require a shared file system between multiple nodes. This tutorial shows how to configure a shared [Kubernetes Persistent Volumes](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#access-modes) (AccessMode `ReadWriteMany` or `RWX`) using [OVHcloud Enterprise File Storage](/links/storage/enterprise-file-storage) as a storage backend.
+
+## Requirements
+
+This tutorial assumes that you already have a working [OVHcloud Managed Kubernetes](/links/public-cloud/kubernetes) cluster, and some basic knowledge of how to operate it. If you want to know more on those topics, please look at the [deploying a Hello World application](/pages/public_cloud/containers_orchestration/managed_kubernetes/deploying-hello-world) documentation.
+
+It also assumes you have an OVHcloud Enterprise File Storage already available. If you don't, you can [order one in the OVHcloud Control Panel](https://www.ovh.com/manager/#/dedicated/netapp/new).
+
+You also need to have [Helm](https://docs.helm.sh/) installed on your workstation, please refer to the [How to install Helm on OVHcloud Managed Kubernetes Service](/pages/public_cloud/containers_orchestration/managed_kubernetes/installing-helm) tutorial.
+
+## Instructions
+
+### Step 1 - Creating a partition and granting your Managed Kubernetes Service access to it
+
+Your Enterprise File Storage can expose multiple partitions, and supports a variety of protocols. Each partition is accessible only from a specific range of IPs. We will create one exposing EFS and make it accessible from your Kubernetes worker nodes.
+
+Access the UI for OVHcloud Enterprise File Storage by clicking the *Storage and backups* then *Enterprise File Storage* menu in the [Bare Metal Cloud section of the OVHcloud Control Panel](https://www.ovh.com/manager/dedicated)
+
+Click on your EFS, then on the `Volumes`{.action} tab, then on the `Create a volume`{.action} button and create the new EFS volume with the following content:
+
+![Create an EFS partition](images/create-efs-volume.png){.thumbnail}
```

Comment:
You should write down the settings to configure.
Usually, images are supporting the text, they shouldn't be mandatory to understand the guide.